src/components/FilterTagsRender.js

```diff
@@ -0,0 +1,37 @@
+import { useHistory, useLocation } from "react-router-dom";
+import { stringify, parse } from "../utilities/queryString";
+import { toggleValueInArray } from "../utilities/filters/toggleValueInArray";
+import FilterTag from "./FilterTag";
+
+export default function FilterTagsRender({ filterPage, filterTags }) {
+  const location = useLocation();
+  const history = useHistory();
+  const parsedQueryString = parse(location.search);
+  const selectedFilters = parsedQueryString[filterPage] || [];
+
+  function handleFilterClick(name) {
+    const newFilters = toggleValueInArray(selectedFilters, name);
+    const parsedOldQueryString = parse(location.search);
+    const newQueryString = stringify({
+      ...parsedOldQueryString,
+      [filterPage]: newFilters,
+    });
+    history.replace({
+      ...location,
+      search: newQueryString,
+    });
+  }
+
+  function isTagToogled(tagName) {
```

Comment:
Toggled, rather than Toogled. Even though, I like the sound of toogled a lot more